test/onPort.test.js

```diff
@@ -11,6 +11,7 @@ test('--on-port flag', { skip: !hasAsyncHooks() }, (t) => {
     /Running 1s test @ .*$/,
     /10 connections.*$/,
     /$/,
+    /$/,
```

Comment:
why?

Reply:
I had to add a [`\n` to the output](https://github.com/mcollina/autocannon/pull/290/files#diff-672f2640e048f0652357caf71e3a69e782159963c981fb5677cd06457e57d5b8R18) to make sure the result table isn't in the same line as spinner.

Reply:
We should really avoid this.